### DISCUSSION TITLE
How do i make a word by word transcription like this

### DISCUSSION THREAD

**Discussion by willepalmberg** [linto-ai/whisper-timestamped#152] [Q&A]
```
1
00:00:00,000 --> 00:00:00,749
Hello

2
00:00:00,749 --> 00:00:01,199
my

3
00:00:01,199 --> 00:00:01,976
name

4
00:00:01,976 --> 00:00:02,203
is

5
00:00:02,203 --> 00:00:03,127
willeishere
```

Thanks!

**Comment by Jeronymous** [comment]
There is a script `make_subtitles` that comes with whisper-timestamped.
It takes the json output from whisper-timestamped and converts that to subtitle formats.
If you use it with `--max_length 1`, you will have segments with one word per segment

**Reply by willepalmberg** [reply]
Oh thanks, can you give an example?

**Reply by Jeronymous** [reply]
`make_subtitles -h`

**Reply by szerwony** [reply]
So `--max_length` can't be specified within whisper_timestamped and it's always needed to manually run `make_subtitles` script after creating subtitles? I would like to create segments with two words.

**Reply by Jeronymous** [reply]
Is it really annoying to run 2 scripts instead of 1? Or what is the problem?
I prefer to keep things separated : Whisper transcription is a costly process, that produces transcriptions in a "canonical" form. The normalization of the transcription (which is not reversible) can be separated.

It's also possible to make a unique script based on python functions, for those who wants to optimize a particular process.
